<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>power</class>
 <widget class="QWidget" name="power">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytPwr">
   <property name="geometry">
    <rect>
     <x>-8</x>
     <y>0</y>
     <width>811</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytPwr
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytPwr.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmmndBtn">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>11</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/cmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sttsBtn">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>11</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/sttsBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnBtn">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>11</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstBtn">
   <property name="geometry">
    <rect>
     <x>628</x>
     <y>11</y>
     <width>154</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingBtn">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>375</y>
     <width>154</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clrBtn">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>375</y>
     <width>154</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/clrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tempClrBtn">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>375</y>
     <width>154</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tempClrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/tempClrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckBtn">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>375</y>
     <width>154</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="inverterSttsBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>230</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inverterSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="offSttsBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>196</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#offSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="UrbanElecGenSttsBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#UrbanElecGenSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upsSttsBtn">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>183</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upsSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="directSttsBtn">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#directSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upsDetailBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upsDetailBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upsDetailBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="chOfBatLbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>225</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#chOfBatLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="invDetailBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>151</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#invDetailBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/invDetailBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="drvDetailBtn">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>270</y>
     <width>148</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#drvDetailBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/drvDetailBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="carSttsBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>175</y>
     <width>64</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#carSttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/carOffSttsBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="carSttsLbl">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(50, 50, 50)</string>
   </property>
   <property name="text">
    <string>Car is OFF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
